src/hotspot/share/runtime/synchronizer.cpp

```diff
@@ -407,10 +407,6 @@ bool ObjectSynchronizer::quick_enter_legacy(oop obj, BasicLock* lock, JavaThread
     return false;  // Slow path
   }
 
-  if (LockingMode == LM_LIGHTWEIGHT) {
-    return LightweightSynchronizer::quick_enter(obj, lock, current);
-  }
-
```

Comment:
This looks unrelated.

Reply:
okay I moved it to another trivial change.  The reason for this cleanup is that this code was in lockStack.inline.hpp indicating it's somewhat performance critical.  In trying to find a performance problem, I don't want to look at this function.